### PR TITLE
Added mychevy docs

### PR DESCRIPTION
**Pull request by sdague**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11409

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/mychevy.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "MyChevy"
+description: "Instructions on how to integrate Chevy Bolt car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
```
Comment (frenck) [inline source/_components/mychevy.markdown]: Incorrect logo

File: source/_components/mychevy.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ha_category: Hub
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+The `MyChevy` component communicates with the
+[my.chevrolet](https://my.chevrolet.com) website using selenium to log
+in as a your user, and screen scrape the data provided. GM does not
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `in as a your user`  -> `in as your user`

File: source/_components/mychevy.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+make it easy to sign up for any official development program, so this
+provides a work around to get access to your data.
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `work around` -> `workaround`

File: source/_components/mychevy.markdown [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+To use MyChevy in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+mychevy:
+  username: email
+  password: password
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/mychevy.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/mychevy.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+information back from the car. As such the mychevy component only
+polls every 30 minutes to not overwhelms that connection.
+
+The OnStar network (or more specifically the gateway used by the
+my.chevrolet website) appears to suffer more than most networks when
+the car is a) in a garage, and b) it's cold outside (like < 15 degrees
+F). One of the provided sensors is a status sensor which indicates if
+we got connectivity with the car on the last polling cycle or not.
+
+The "API" for this is written by webscraping. As such, it only
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `webscraping` -> `web scraping`

File: source/_components/mychevy.markdown [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+currently is known to work if you have a Chevy Bolt EV, and only 1
+chevy car connected to OnStar. Patches for extended support should go
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `chevy` -> `Chevy`

File: source/_components/mychevy.markdown [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+to the https://github.com/sdague/mychevy project first, then
+home-assistant can be extended.
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `home-assistant` -> `Home Assistant`

**Comment by frenck** [comment]
@sdague Could you please take a look at my review comments? 👍

**Comment by sdague** [comment]
ACK. Thank you for the review and catching those issues.

File: source/_components/mychevy.markdown (hunk 1 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 2 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 3 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 4 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 5 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 6 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 7 above)
Comment (sdague) [reply source/_components/mychevy.markdown]: ack

File: source/_components/mychevy.markdown (hunk 4 above)
Comment (frenck) [reply source/_components/mychevy.markdown]: @sdague You still need to remove this header...

File: source/_components/mychevy.markdown [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+title: "MyChevy"
+description: "Instructions on how to integrate Chevy Bolt car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: chevy.png
+ha_category: Hub
+ha_release: 0.61
```
Comment (frenck) [inline source/_components/mychevy.markdown]: `0.61` is already shipped. Please update this to `0.62`

**Comment by frenck** [comment]
Can be merged as soon as the build passes.

**Pull request merged**